examples/helm-blue-green/README.md

```diff
@@ -0,0 +1,30 @@
+# Blue Green
```

Comment:
Feel like this doc is more related to argocd. Would be better if we add some description of argo-rollout in the context.

Reply:
Yes I took that example from ArgoCD. Ok I will update the Readme.